test/features/icons/icons.test.js

```diff
@@ -27,34 +27,120 @@ describe('Icon Suppprt', () => {
   });
 
   before(async () => {
+    document.body.innerHTML = await readFile({ path: './mocks/body.html' });
     icons = document.querySelectorAll('span.icon');
     await loadIcons(icons, config);
-    await loadIcons(icons, config); // Test duplicate icon not created if run twice
   });
 
   it('Fetches successfully with cache control enabled', async () => {
     const otherIcons = [createTag('span', { class: 'icon icon-play' })];
+    document.body.appendChild(otherIcons[0]);
+
     await loadIcons(otherIcons, config);
+    const svg = otherIcons[0].querySelector('svg');
+    expect(svg).to.exist;
   });
 
-  it('Replaces span.icon', async () => {
+  it('Replaces span.icon', () => {
     const selector = icons[0].querySelector(':scope svg');
     expect(selector).to.exist;
   });
 
-  it('No duplicate icon', async () => {
+  it('No duplicate icon', () => {
     const svgs = icons[0].querySelectorAll(':scope svg');
```

Comment:
You'd want to run `await loadIcons(icons, config);` again here, given before it was ran twice, no?